core/src/main/scala/com/dimafeng/testcontainers/DockerComposeContainer.scala

```diff
@@ -109,6 +121,7 @@ object DockerComposeContainer {
 }
 
 class DockerComposeContainer(composeFiles: ComposeFile,
+                             services: Services = Services.All,
```

Comment:
Can we move this to the last parameter? (in case somebody uses `exposedServices` positionally)

Reply:
Thank you, I've fixed it.